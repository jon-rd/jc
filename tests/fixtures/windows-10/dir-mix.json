[{"date":"03/25/2021","time":"02:38 PM","dir":false,"size":4253,"filename":"docgen.sh","parent":"C:\\Users\\Yoda\\Desktop\\jc","epoch":1616708280},{"date":"03/25/2021","time":"02:36 PM","dir":true,"size":null,"filename":".","parent":"C:\\Users\\Yoda\\Desktop\\jc\\docs","epoch":1616708160},{"date":"03/25/2021","time":"02:36 PM","dir":true,"size":null,"filename":"..","parent":"C:\\Users\\Yoda\\Desktop\\jc\\docs","epoch":1616708160},{"date":"03/25/2021","time":"02:38 PM","dir":true,"size":null,"filename":"parsers","parent":"C:\\Users\\Yoda\\Desktop\\jc\\docs","epoch":1616708280},{"date":"03/25/2021","time":"02:36 PM","dir":false,"size":2713,"filename":"readme.md","parent":"C:\\Users\\Yoda\\Desktop\\jc\\docs","epoch":1616708160},{"date":"03/25/2021","time":"02:36 PM","dir":false,"size":1338,"filename":"utils.md","parent":"C:\\Users\\Yoda\\Desktop\\jc\\docs","epoch":1616708160},{"date":"03/25/2021","time":"02:38 PM","dir":false,"size":58995,"filename":"EXAMPLES.md","parent":"C:\\Users\\Yoda\\Desktop\\jc","epoch":1616708280}]
