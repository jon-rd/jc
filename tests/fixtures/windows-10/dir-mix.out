 Volume in drive C has no label.
 Volume Serial Number is 246B-3B5A

 Directory of C:\Users\Yoda\Desktop\jc

03/25/2021  02:38 PM             4,253 docgen.sh
               1 File(s)          4,253 bytes

 Directory of C:\Users\Yoda\Desktop\jc\docs

03/25/2021  02:36 PM    <DIR>          .
03/25/2021  02:36 PM    <DIR>          ..
03/25/2021  02:38 PM    <DIR>          parsers
03/25/2021  02:36 PM             2,713 readme.md
03/25/2021  02:36 PM             1,338 utils.md
               2 File(s)          4,051 bytes

 Directory of C:\Users\Yoda\Desktop\jc

03/25/2021  02:38 PM            58,995 EXAMPLES.md
               1 File(s)         58,995 bytes
               0 Dir(s)  34,115,264,512 bytes free
